### PR TITLE
add auth element

### PR DESCRIPTION
**Pull request by locomuco**
adds an auth element to separate between data and authentication

### CODE REVIEW

**Comment by locomuco** [comment]
I signed it!

**Comment by robdodson** [comment]
Sorry ti's taken so long for me to review this, I've been on vacation :) Looking it over now.

**Comment by locomuco** [comment]
no problem, not in a hurry :+1:

File: app/elements/todo-auth/todo-auth.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  </template>
+  <script>
+    Polymer({
+
+      is: 'todo-auth',
+
+      properties: {
+
+        user: {
+                notify: true
```
Comment (robdodson) [inline app/elements/todo-auth/todo-auth.html:90]: I think this should be 2 spaces. Seems like it's maybe 4 here?

**Comment by robdodson** [comment]
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

**Comment by robdodson** [comment]
Cool I fixed up the formatting issue and did a squash commit. https://github.com/PolymerLabs/todo-list/commit/d1ba1b94a20910079f0a2eb1893164a5d287f501

Thank you!

**Pull request closed without merging**